### ISSUE TITLE
Question: Is it possible to change file size limit? "Request Entity Too Large"

### ISSUE THREAD

**Issue by WychWitch** [rr-/szurubooru#217]
Hello again! I finally have my szurubooru system working flawlessly! However, I've encountered a problem where quite a few of my images are simply too large to upload, throwing the message `"Request Entity Too Large"` on the client side (nothing as far as I can tell on the server-side).

I understand why this is in place for public boorus, but since this is a private one I'd like to bump up the file size rather than downgrading my images. I've looked in the configs (config.yaml, both docker files and the .env) and a few .py files but I couldn't find anything setting the file size limit.

Is there a way to change the file size limit? Or is this some restriction of a program and not something that can be changed?

Thank's again in advance!

**Comment by ReAnzu** [comment]
The only limitation on upload size that I’m aware of in the docker configuration is in the client/nginx.conf.docker where the following line

‘’’
client_max_body_size 100M;
‘’’

Sets the max upload to a 100MB. If you’re using the docker setup, you can change that line in the client/nginx.conf.docker and the do a 

‘’’
docker-compose build client && docker-compose down && docker-compose up
‘’’

to deploy the new nginx with the updated config.

If you’re not using docker, you need to go and update your nginx config manually (If you’re using Apache instead, there’s a similar client body size directive, but I don’t know it off the top of my head).

**Comment by WychWitch** [comment]
Huh, this is weird.

Not only did it not work when set to 1000M, but when I checked the file it's only 1.51mb in size, far bellow the initial 100M limit.

Then I thought it might be the resolution, but is only 1000x1015 where the first post I made successfully was 2048x1008 (though its size was considerably less at 220kb)

I'm not sure what's going on? Maybe there's another config somewhere, I'll keep looking as well. Thanks still!

**Comment by WychWitch** [comment]
OH nevermind I was being dumb hahaha. I had it running through another nginx on the host machine and I completely forgot! I added the line `client_max_body_size 1000M;` to the top of the http block in my `/etc/nginx/nginx.conf` file and it immediately worked! Turns out nginx has a default max of 1MB (which explains why the 1.51mb file couldn't upload, despite the fact that it should've worked in the dockerfile).

Thanks so much @ReAnzu!
